### ISSUE TITLE
Tire Load too high Increase tire pressure on aquatrans 7300S, 

### ISSUE THREAD

**Issue by andycr13** [RealismusModding/FS17_RM_soilCompaction#43]
when filling the vanilla water trailer, Aquatrans 7300S with water, warning comes up saying "Tire Load too high Increase tire pressure on aquatrans 7300S!" ive tried to lower and increase the pressure, still the warning remains and reduces speed to 7mph. doesnt seem to have a happy tire pressure where it allows the trailer to function correctly
![fsscreen_2018_10_16_14_49_37](https://user-images.githubusercontent.com/44201993/47021126-bd677e80-d152-11e8-8982-14c02d3228dd.png)
i get the same issue on vanilla map with only soil compaction mod selected

**Comment by andycr13** [comment]
once the water level reaches between 90 and 95% full. this warning comes up and doesnt leave. hope this is helpful to someone

**Comment by reallogger** [comment]
Yes this is helpful. Thanks for reporting. This function clearly needs some further adjustment..

**Comment by andycr13** [comment]
perfect thank you. i love the mod so if i can help improve i will

**Comment by andycr13** [comment]
getting the same warning with the square bale auto load trailer from the vanilla game when fully loaded
.

**Comment by moep0710** [comment]
Just a hint for everybody who got problems with the tirepressure-function: Just replace your scTirePressure with the version from Aug 20th, it will deaktivate the speed limit:
https://github.com/RealismusModding/FS17_RM_soilCompaction/blob/1d541fdbb775f08b1396d2820eb734ad627849d9/src/vehicles/scTirePressure.lua
